examples/ls/main.go

```diff
@@ -0,0 +1,22 @@
+/*Example for ListFiles function.
+`ls` binary will list content of directory provided via args
+
+Example of runing this binary at `examples/` directory:
+./examples/ls/ls examples/
+examples/cat
+examples/cat2
+examples/echo
+examples/grep
+
+*/
+package main
+
+import (
+	"os"
+
+	"github.com/bitfield/script"
+)
+
+func main() {
+	script.ListFiles(os.Args[1]).Stdout()
```

Comment:
We need to handle the case where there are no command-line arguments.

Reply:
Also, I think we need to handle the special cases of `.` and `..`, which Unix `ls` does.

Reply:
i've fixed ls example to hide files starting with single dot. But i don't understand how do you want me to handle double dot? What is special with it?